android/src/main/java/com/rudderstack/android/internal/RudderPreferenceManager.kt

```diff
@@ -109,12 +110,12 @@ internal class RudderPreferenceManager(application: Application,
     val userId: String?
         get() = preferences.getString(RUDDER_USER_ID_KEY.key, null)
 
-    fun saveOptStatus(optStatus: Boolean) {
-        preferences.edit().putBoolean(RUDDER_OPT_STATUS_KEY.key, optStatus).apply()
-    }
-
-    val optStatus: Boolean
-        get() = preferences.getBoolean(RUDDER_OPT_STATUS_KEY.key, false)
+//    fun saveOptStatus(optStatus: Boolean) {
+//        preferences.edit().putBoolean(RUDDER_OPT_STATUS_KEY.key, optStatus).apply()
+//    }
+//
+//    val optStatus: Boolean
+//        get() = preferences.getBoolean(RUDDER_OPT_STATUS_KEY.key, false)
```

Comment:
I guess this is going to be used in the next small feature. 👀 
If so let's keep it!

Reply:
Removing this, will be added in different PR